### PR TITLE
removed the test object from modules view mode

### PR DESCRIPTION
**Pull request by joenan**
removed the test object from modules ``view mode`` 
```
"test": {
	"compType": "string",
	"comp": "255"
}

```

The test object was removed from both the inputs and output arrayss
`data.applicationDSL.ui.comp.io.inputs[]`
`data.applicationDSL.ui.comp.io.outputs[]`

### CODE REVIEW

File: server/api-service/lowcoder-server/src/main/resources/application-lowcoder.yml
Comment (FalkWolsky) [inline server/api-service/lowcoder-server/src/main/resources/application-lowcoder.yml:1]: application-lowcoder.yml needs to be on the ignore list

**Review by FalkWolsky — CHANGES_REQUESTED** [review]
Please put application-lowcoder.yml to gitignore.
Rest is super.

File: server/api-service/lowcoder-server/src/main/resources/application-lowcoder.yml
Comment (joenan) [reply server/api-service/lowcoder-server/src/main/resources/application-lowcoder.yml:1]: This is now done. Thank you

**Review by FalkWolsky — DISMISSED** [review]
That looks good now. Thank you

**Review by FalkWolsky — APPROVED** [review]
Hope, this time the commit comment passes the test

**Pull request merged**